bindata/network/ovn-kubernetes/common/002-rbac.yaml

```diff
@@ -24,16 +24,27 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
ovnkube-node no longer needs access to patch `nodes` and `pods` right? the RBAC still has it:
https://github.com/openshift/cluster-network-operator/blob/f68d528042f49bfd91f3a07588c5cea656cdbffd/bindata/network/ovn-kubernetes/common/002-rbac.yaml#L100-L108
https://github.com/openshift/cluster-network-operator/blob/f68d528042f49bfd91f3a07588c5cea656cdbffd/bindata/network/ovn-kubernetes/common/002-rbac.yaml#L55-L63

Reply:
removed the permissions